### PR TITLE
fix(frontend): dashboard card summary breakdown UI

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/About.vue [hunk 1]
```diff
@@ -90,10 +90,10 @@ const remoteAboutLogo =
 </script>
 
 <template>
-  <VCard class="pb-6" width="500px" light :class="css.about">
-    <div class="pt-6 pb-3 text-h2 font-black white--text primary">
-      <span class="px-6">{{ t('app.name') }}</span>
-      <span class="d-block mb-3 pl-6 text-caption">
+  <VSheet>
```
Comment (kelsos) [inline frontend/app/src/components/About.vue:93]: Do you think it's possible to do the same with `RuiCard`?

File: frontend/app/src/components/dashboard/summary-card/SummaryCard.vue [hunk 2]
```diff
@@ -28,10 +28,10 @@ const slots = useSlots();
   <VCard
     :loading="isLoading"
     :class="`dashboard__summary-card__${name}`"
-    class="pb-1"
+    class="pb-4"
```
Comment (kelsos) [inline frontend/app/src/components/dashboard/summary-card/SummaryCard.vue:31]: Also here do you think it's possible to migrate to RuiCard?

Comment (lukicenturi) [reply frontend/app/src/components/dashboard/summary-card/SummaryCard.vue]: updated

File: frontend/app/src/components/status/sync/SyncButtons.vue [hunk 3]
```diff
@@ -24,7 +24,7 @@ const action = (action: SyncAction) => {
 
 <template>
   <div class="flex flex-row justify-between gap-1">
-    <RuiTooltip open-delay="400" class="w-full" close-delay="50000">
+    <RuiTooltip open-delay="400" class="w-full">
```
Comment (lukicenturi) [inline frontend/app/src/components/status/sync/SyncButtons.vue:27]: I'd like to know who responsible for this.🤪

Comment (kelsos) [reply frontend/app/src/components/status/sync/SyncButtons.vue:27]: @lukicenturi hahahaha hmm probably me, if you do git-blame I am pretty sure my name is there.

I tried something and I supposedly reverted it to remove the close-delay...

**Comment by lukicenturi** [comment]
This can be checked again @kelsos

File: frontend/app/src/components/About.vue [hunk 4]
```diff
@@ -87,153 +87,171 @@ const versionText = computed(() => {
...
+                    <template #activator="{ on, attrs }">
+                      <VBtn
+                        v-bind="attrs"
+                        icon
+                        x-small
+                        v-on="on"
+                        @click="openPath(dataDirectory)"
+                      >
+                        <VIcon x-small>mdi-launch</VIcon>
+                      </VBtn>
```
Comment (kelsos) [inline frontend/app/src/components/About.vue:154]: Do What do we need to also migrate these?

File: frontend/app/src/components/About.vue [hunk 5]
```diff
@@ -87,153 +87,171 @@ const versionText = computed(() => {
...
-    </VCardText>
-  </VCard>
+                  </template>
+                  <span :class="css.directory">
+                    {{ dataDirectory }}
+                  </span>
+                </RuiTooltip>
+                <div v-if="!isPackaged" class="ml-2">
+                  <RuiTooltip :popper="{ placement: 'top' }" open-delay="400">
+                    <template #activator="{ on, attrs }">
```
Comment (kelsos) [inline frontend/app/src/components/About.vue:145]: Are the `on, attrs` needed with `RuiTooltip`?
I have been removing them and everything seems to be fine.

**Comment by kelsos** [comment]
ok so other than the two questions above I don't see anything wrong.

**Review by kelsos — APPROVED** [review]
let's merge

**Pull request merged**